readthedocs/core/forms.py

```diff
@@ -41,6 +39,7 @@ def save(self, commit=True):
             user.first_name = first_name
             user.last_name = last_name
             user.save()
+            update_change_reason(user, 'Changed from: form')
```

Comment:
This is kind of an special case, because this is a model form for UserProfile, not User, so we can't use inheritance like the other forms.

Reply:
This should be a code comment - useful context 👍